lib/msal-browser/docs/login-user.md

```diff
@@ -97,7 +97,13 @@ These APIs will return an account object or an array of account objects with the
 
 ## Silent login with ssoSilent()
 
-If you already have a session that exists with the authentication server, you can use the ssoSilent() API to make request for tokens without interaction. You will need to pass a `login_hint` (which can be retrieved from the account object `username` property or the `upn` claim in the ID token) in the request object in order to successfully obtain a token silently.
+If you already have a session that exists with the authentication server, you can use the ssoSilent() API to make request for tokens without interaction. You will need to pass one of the following into the request object in order to successfully obtain a token silently.
+
+- `login_hint` (which can be retrieved from the account object `username` property or the `upn` claim in the ID token)
```

Comment:
can we please reverse this order? For perception of precedence.

Reply:
so account, sid, login_hint?

Reply:
done.